info.md

```diff
@@ -21,7 +21,7 @@ Open that directory and add a file called **gamestate_integration_homeassistant.
 Add the following content and replace the "uri" with the one copied during the integration setup:
 
 ```
-"HomeAssistant Integration v.1"
+"HomeAssistant Integration v.2"
```

Comment:
Please change to v0.2 as you defined in the manifest

Reply:
Good catch. I only later discovered your release versioning, so manifest, changelog and and this cfg now all follow the SemVer scheme.